src/analysis/class_builder.rs

```diff
@@ -98,14 +99,21 @@ fn analyze_property(
     if !for_builder {
         return None;
     }
-    if let Ok(ref s) = used_rust_type(env, prop.typ, false) {
+    let type_str = used_rust_type(env, prop.typ, false);
+    if let Ok(ref s) = type_str {
         if !s.contains("GString") {
             imports.add_used_type_with_version(s, prop_version);
```

Comment:
Please add [`set_defaults`](https://github.com/gtk-rs/gir/blob/master/src/analysis/properties.rs#L111) before this if, and just `add_used_type`, then `imports.add("glib::object::IsA");` will have same version